Apps/Examples/Examples/Assets.xcassets/user_puck_icon.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Want to confirm that missing "preserving vector data" was intentional with SVG image.

Reply:
It was not intentional -- I updated to include